### PR TITLE
use new JAVA_HOME path and *-default-jvm for jdk/jre-11

### PR DESCRIPTION
**Pull request by joshrwolf**
Brings 11 up to the new standard installation path for java, and uses the `-default-jvm` package.

Related: https://github.com/wolfi-dev/os/pull/2075

### CODE REVIEW

**Comment by joshrwolf** [comment]
draft until jdk11 packages are updated

File: images/jre/configs/openjdk-11.apko.yaml
Comment (patflynn) [inline images/jre/configs/openjdk-11.apko.yaml:23]: could this be breaking?

**Comment by patflynn** [comment]
the package is updated already isn't it? https://github.com/wolfi-dev/os/pull/2075

File: images/jre/configs/openjdk-11.apko.yaml
Comment (joshrwolf) [reply images/jre/configs/openjdk-11.apko.yaml:23]: potentially yeah, if folks are using absolute paths within their build environment. do we have precedence for these kind of changes?

**Comment by joshrwolf** [comment]
> the package is updated already isn't it? [wolfi-dev/os#2075](https://github.com/wolfi-dev/os/pull/2075)

it was missing aarch64, which should be g2g now, bringing this back as ready

File: images/jre/configs/openjdk-11.apko.yaml
Comment (patflynn) [reply images/jre/configs/openjdk-11.apko.yaml:23]: I'll defer to @imjasonh on making a call on this.

Comment (patflynn) [reply images/jre/configs/openjdk-11.apko.yaml:23]: maybe move to a different PR or is this related to the jdk refactoring?

Comment (kaniini) [reply images/jre/configs/openjdk-11.apko.yaml:23]: Not related.  `/home/build` is just the default build workspace in melange, but does not matter here, except for scenarios where we might want it to be consistent for education reasons.

File: images/jdk/configs/openjdk-11.apko.yaml
Comment (patflynn) [inline images/jdk/configs/openjdk-11.apko.yaml:11]: based on what I heard from @kaniini we just need the openjdk-11-default-jvm package as it pulls in openjdk-11 as a runtime dep.

Comment (patflynn) [reply images/jdk/configs/openjdk-11.apko.yaml:11]: actually I think I misunderstood. hold off on that change

Comment (patflynn) [reply images/jdk/configs/openjdk-11.apko.yaml:11]: I think my understanding now is that -default-jvm only pulls in the JRE and not the JDK. So perhaps we keep it here but remove from the JRE image?  I guess it would have no impact either way because it's a subset or equivalent?

**Review by patflynn — APPROVED** [review]

**Pull request merged**
